openslides/users/auth.py

```diff
@@ -1,46 +1,12 @@
 from django.contrib.auth import get_user as _get_user
-from django.contrib.auth import get_user_model
-from django.contrib.auth.backends import ModelBackend
 from django.contrib.auth.models import AnonymousUser as DjangoAnonymousUser
 from django.contrib.auth.models import Permission
-from django.db.models import Q
 from django.utils.functional import SimpleLazyObject
 from rest_framework.authentication import BaseAuthentication
 
 from ..core.config import config
 
 
-# Registered users
-
-class CustomizedModelBackend(ModelBackend):
-    """
-    Customized backend for authentication. Ensures that registered users
-    have all permissions of the group 'Registered' (pk=2). See
-    AUTHENTICATION_BACKENDS settings.
-    """
-    def get_group_permissions(self, user_obj, obj=None):
```

Comment:
I think we still need this. Test it: Create a new user without any group. Try to login with this user. Looks like this user has no group and no permissions. Expected: Users without any group gets all permissions of the default (guest) group.